src/test/fuzz/miniscript.cpp

```diff
@@ -1017,7 +1148,7 @@ void TestNode(const NodeRef& node, FuzzedDataProvider& provider)
             assert(it != TEST_DATA.dummy_sigs.end());
             return it->second.second;
         }
-        case Fragment::MULTI: {
+        case Fragment::MULTI: case Fragment::MULTI_A: {
```

Comment:
Nit: missing a newline

Reply:
Hmm i thought i was following the style used elsewhere, but i must have dreamed it. Done.